### DISCUSSION TITLE
Which firmware for my yi camera 1080p outdoor?

### DISCUSSION THREAD

**Discussion by miloit** [alienatedsec/yi-hack-v5#365] [Q&A]
hello,
I have a yi camera 1080p
Model YHS,3017
SN: 9CUSYCOX89
FCC-ID 2AFIB-YHS3017
Firmware is 3.0.0.0D_201809111054

Which version of the file do i need?

**Comment by alienatedsec** [comment]
Follow the latest release notes and your camera is `yi_outdoor` so should be supported.
